### PR TITLE
Fixing confusing translation re logfile data

### PR DESCRIPTION
**Pull request by philipecosia**
Improving after getting feedback from @LostCrew here https://github.com/ecosia/ecosia-privacy-policy/pull/5

### CODE REVIEW

File: en.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In the case of use of the website for purely informational purposes, if you do n
  - Operating system and its surface
  - Language and version of the browser software.
 
-The data are saved in the logfiles of our system. These data and other personal data of the user are not saved. The data are deleted once they are no longer required to achieve the purpose of your gathering. In the event that the data are saved in log files, this occurs at the most seven days later. 
+The data are saved in the logfiles of our system. These data can never be connected to other personal data of our users. The data are deleted once they are no longer required to achieve the purpose of your gathering. In the event that the data are saved in log files, this occurs at the most seven days later.
```
Comment (m90) [inline en.md]: Shouldn't this read `This data can never ...`

**Comment by philipecosia** [comment]
OK should be fixed now

**Review by m90 — APPROVED** [review]
:1st_place_medal:

**Comment by LostCrew** [comment]
GOGOGO

**Review by knee-guts — APPROVED** [review]
🥗

**Pull request merged**